libs/template/templates/default-python/template/{{.project_name}}/dev/exploration.py

```diff
@@ -0,0 +1,6 @@
+
+# Databricks notebook source
```

Comment:
why do we have both ipynb and a python source notebook?

Reply:
Everything will move to ipynb now that it's GA.